docker/README.md

```diff
@@ -96,3 +105,26 @@ mkdir -p /CodeCompass/workspace
   -d "sqlite:" \
   -w /CodeCompass/workspace
 ```
+
+# Deployment
+
+## Build image for web
+Build the web environment image from CodeCompass `master` branch:
+```
+docker build -t codecompass-web --no-cache docker/web
+```
+
+See more information [below](#how-to-run-codecompass-webserver-in-docker) how
+to use this image to start a CodeCompass webserver.
+
+## How to run CodeCompass webserver in docker
+You can use the `codecompass-web` image created
+[above](#build-image-for-web) to start a CodeCompass webserver.
+For this run the following command:
+```bash
+docker run \
+  --volume /home/ecsomar/workspace/codecompass/:/workspace \
```

Comment:
Are you sure about this user name? :) At other parts of this documentation we used `/path/to/host/CodeCompass/workspace`.